### PR TITLE
Missing keywords for tab completion

### PR DESCRIPTION
**Pull request by mkopec3**
Hi,

Just found out, that there are many missing keywords in ***radare_argv[]**, which are used for tab completion purpose.
Curious, whether keywords are limited intentionally (e.g. to highlight only most important commands), or it's something worth to fix it (by adding those missed)?

### CODE REVIEW

**Comment by radare** [comment]
That's maintained by hand, so it's not in sync with all the commands, feel free to add more :)

thanks!

**Comment by radare** [comment]
Can you change the status of this pr to not be a draft? Otherwise i cant merge it

**Comment by mkopec3** [comment]
Didn't finish this PR yet, so far added only keywords starting with d & e + rearranged their order for better clarity. Still there're many commands missing...
I will let you know here, when PR is ready to review & merge

**Comment by radare** [comment]
Ok

**Comment by radare** [comment]
Btw its preferibly to rebase than keep merging and create more commits in the branch.

**Comment by mkopec3** [comment]
Thanks for the feedback.
I definitely have too much mess in my forked repo :(
Still adding additional keywords... great r2 learning experience :)

**Comment by mkopec3** [comment]
PR ready to review & merge.
I'm also gonna review **init_autocomplete()** soon...

**Comment by radare** [comment]
thanks!

File: libr/core/core.c [hunk 1]
```diff
@@ -814,50 +814,195 @@ R_API RCore *r_core_new() {
...
-	"z", "z*", "zj", "z-", "z-*",
-	"za", "zaf", "zaF",
-	"zo", "zoz", "zos",
-	"zfd", "zfs", "zfz",
-	"z/", "z/*",
+	"whereis", "which", "ls", "rm", "mkdir", "pwd", "cat", "less",
+	"#?", "#!", "#sha1", "#crc32", "#pcprint", "#sha256", "#sha512", "#md4", "#md5",
+	"#!python", "#!perl", "#!vala", "#!pipe",
+	"*?", "*",
+	"(", "(*", "(-", "()", ".?", ".", "..", "...", ".:", ".--", ".-", ".!", ".(", "./",
```
Comment (radare) [inline libr/core/core.c]: yesterday i added .*

File: libr/core/core.c [hunk 2]
```diff
@@ -814,50 +814,195 @@ R_API RCore *r_core_new() {
...
+	".afbr-", ".afbr-*", ".ar*", ".ar-",
```
Comment (radare) [inline libr/core/core.c]: those are not valid commands, the '.' is a suffix modifier. so remove thosoe

**Comment by mkopec3** [comment]
done

**Comment by radare** [comment]
Thanks!

**Pull request merged**